libs/traits/src/lib.rs

```diff
@@ -377,9 +380,10 @@ pub trait TrancheCurrency<PoolId, TrancheId> {
 /// A trait, when implemented allows to invest into
 /// investment classes
 pub trait Investment<AccountId> {
-	type Error: Debug;
-	type InvestmentId;
 	type Amount;
+	type CurrencyId;
+	type Error: Debug;
+	type InvestmentId: Into<Self::CurrencyId>;
```

Comment:
Why is the `Into<Self::CurrencyId>` bound needed here?

Reply:
Same question 👀

Reply:
Good catch, this can just be removed!